app/ante/eth.go

```diff
@@ -429,6 +426,20 @@ func (vbd EthValidateBasicDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simu
 				return ctx, sdkerrors.Wrapf(sdkerrors.ErrUnknownRequest, "invalid message type %T, expected %T", msg, (*evmtypes.MsgEthereumTx)(nil))
 			}
 
+			// Validate Size_ field, should be kept empty
+			if msgEthTx.Size_ != 0 {
```

Comment:
why not check these fields on the `msg.ValidateBasic` since they are stateless?

Reply:
makes sense, it didn't run `msg.ValidateBasic` at all before, I've just added that.

Reply:
Not sure if not running `msg.ValidateBasic` is expected: https://github.com/cosmos/cosmos-sdk/issues/12607